authserver/token_test.go

```diff
@@ -23,19 +23,14 @@ func init() {
 	clientSecretKey := make([]byte, 32)
 	rand.Read(clientSecretKey)
 
-	privateRSAKey, _ := rsa.GenerateKey(rand.Reader, 2048)
-	privateRSAJWK := jose.JSONWebKey{
-		Key:       privateRSAKey,
-		KeyID:     string(jose.RS256),
-		Algorithm: string(jose.RS256),
-		Use:       "sig",
-	}
+	privateKey, _ := ecdsa.GenerateKey(elliptic.P256(), rand.Reader)
```

Comment:
Hmm but this is in the `init` function so I don't have access to `t`

Reply:
Good point. In this case, I recommend `panic`. At least if something errors (like permission denied trying to open `/dev/urandom`) you get some sort of message. And this really _shouldn’t_ ever fail… but it’d be nice to know when it does, instead of a silent apparent success.